### ISSUE TITLE
Ability to set initcwnd initrwnd

### ISSUE THREAD

**Issue by KyleSanderson** [flannel-io/flannel#1780]
## Expected Behavior
MTU is tuneable and gains have been seen.

## Current Behavior
TCP window size is still small. 

## Possible Solution
Expose initcwnd initrwnd on route creation.

## Steps to Reproduce (for bugs)

1. Use Flannel
2.
3.
4.

## Your Environment

* Flannel version: 0.22.0
* Backend used (e.g. vxlan or udp): native
* Etcd version: n/a
* Kubernetes version (if used): v1.24.10+rke2r1
* Operating System and version: 6.4-rc6
* Link to your project (optional):

**Comment by rbrtbnfgl** [comment]
We are not configuring TCP windows size on flannel I think that the linux default behaviour is used.
It's probably configurable on the system with sysctl or directly on the pod that needs to use a different size.

**Comment by KyleSanderson** [comment]
Indeed, it's set on the host, but the netns namespaces don't have the same value set which is tragic. Can we please add the ability to set this value (it's on the routing table).

**Comment by rbrtbnfgl** [comment]
But the value should be configured inside the namespace route too?

**Comment by KyleSanderson** [comment]
That's what's missing presently.

**Comment by rbrtbnfgl** [comment]
I think this could be done by the cni-plugin when Kubelet creates the pod the route on its namespace should be configured with these parameters.

**Comment by KyleSanderson** [comment]
Sorry - I don't understand.

**Comment by KyleSanderson** [comment]
I don't use flannel anymore because I flipped to nixos, and the bridge helps immensely with a fixed crank for internal transfers.
